tests/xtro-sharpie/common-Photos.ignore

```diff
@@ -1 +1,4 @@
 !missing-protocol-conformance! PHFetchResult should conform to NSFastEnumeration
+
+# Enum is named PHPhotosErrorDomain and is clashing with domain name
```

Comment:
add a link to the feedback you filed and maccore _shadow_ issue

Reply:
I'm sorry, what is a maccore shadow issue?

Reply:
when we file a feedback issue with Apple (only you can see it) we add a copy in maccore (so everyone can see it)
https://github.com/xamarin/maccore/issues?q=is%3Aissue+is%3Aopen+label%3Aapple-feedback

Reply:
I was not able to open this link, is it possible that I do not have authorization to this page?

Reply:
yes, you might not have access to the private repo :(